app/src/main/java/org/systers/mentorship/view/activities/LoginActivity.kt

```diff
@@ -36,9 +39,17 @@ class LoginActivity : BaseActivity() {
                     intent = Intent(this, MainActivity::class.java)
                     startActivity(intent)
                     finish()
-                } else {
-                    Snackbar.make(getRootView(), loginViewModel.message, Snackbar.LENGTH_LONG)
-                            .show()
+
+                }  else {
+                    if (loginViewModel.message == getString(R.string.verify_email)) {
+                        Snackbar.make(getRootView(), loginViewModel.message, Snackbar.LENGTH_INDEFINITE)
+                                .setAction(getString(R.string.resend_email)) {
+                                    loginViewModel.resendEmail(Email(tiUsername.editText?.text.toString()))
```

Comment:
Interesting! What would be the fix? Maybe this can be fixed in this PR @m-murad , because @mohak1283 is already working on this feature? What do you think it should happen? Should we ask the user to enter the email, or open a dialog to put the email?

Reply:
Okay so should it be fixed in this PR?

Reply:
I think it can be fixed here. But I don't mind if this PR does the minimum, and then this is fixed elsewhere. What do you think @mohak1283

Reply:
Yes @isabelcosta I agree with you. Let this feature be fixed in separate PR. 👍